mirrors.yml

```diff
@@ -1,15 +1,18 @@
 origin:
-    desc: AOSC main repository (Aperture Science Limited)
+    desc: AOSC main repository with Auto-Redirect
     url: https://repo.aosc.io/anthon/
 origin4:
-    desc: AOSC main repository (Aperture Science Limited, IPv4 only)
+    desc: AOSC main repository with Auto-Redirect (IPv4 only)
     url: https://v4.repo.aosc.io/anthon/
 origin6:
-    desc: AOSC main repository (Aperture Science Limited, IPv6 only)
+    desc: AOSC main repository with Auto-Redirect (IPv6 only)
     url: https://v6.repo.aosc.io/anthon/
 repo-us:
     desc: AOSC main repository (United States)
     url: https://repo-us.aosc.io/anthon/
+repo-hk:
```

Comment:
No separate IPv4 and IPv6 domains for now?

Reply:
> No separate IPv4 and IPv6 domains for now?

nope